spec/features/mailers/submission_mailer_spec.rb

```diff
@@ -1,6 +1,31 @@
 require "rails_helper"
 require "capybara/email/rspec"
 
+feature "Wysłanie maila potwierdzającego przyjęcie zgłoszenia" do
+
+  context "zgłoszenie w bazie" do
+    include_context "admin login"
+
+    background do
+      Person.create!(name: "Andrzej", surname: "Kapusta", email: "a.kapusa@gmail.com", sex: "mężczyzna", roles: ['autor'])
+      Submission.create!(person: Person.first, status: "nadesłany", polish_title: "Artykuł o kotach", english_title: "Story about cats", english_abstract: "Cats are so amazing", english_keywords: "cats", received: "02-03-2016", language: "polski")
+    end
+
+    scenario "sprawdzenie wysłania maila potwierdzającego przyjęcie zgłoszonego tekstu" do
+     clear_emails
+     visit '/submissions'
+     click_link 'Polski tytuł'
+     click_link 'Edytuj'
+     select "nadesłany", from: "Status"
```

Comment:
Nie bardzo wiem jak wejść na widok osoby w takim przypadku. Zaczęłam teraz analizować i po zalogowaniu jako użytkownik mam od razu w menu po lewej stronie "Zgłoś artykuł" czyli public_submission - może jednak przerobić na public_submission tego mailera i wtedy mogłabym w teście od razu wywołać "Zgłoś artykuł" - czy upraszczam i nie odbywa się to w ten sposób?